### ISSUE TITLE
completion not completing anything

### ISSUE THREAD

**Issue by euri10** [DanCardin/cappa#83]
I'm not seeing anything completed when pressing tab after my cli name.

it's entirely possible I'm configuring things badly, here's what I did:

1. write a little cli `toto`, works fine, displays nice looking help etc /.-
2. issue a `toto --completion generate > ~/oh-my=zsh/custom/plugins/toto/toto.plugin.zsh`
3. added my toto to the plugins line in `.zshrc`

am I missing something obvious ?

**Comment by DanCardin** [comment]
That **sounds** right (modulo i assume it should be `~/oh-my-zsh`, not `=zsh` :P).

What does the content of `toto.plugin.zsh` look like, and can you perhaps echo something inside of it to ensure it's actually being executed?

**Comment by euri10** [comment]
ok i tested in bash and it works fine so I might have a zsh config issue not loading the compdef file...wil try harder and report

**Comment by euri10** [comment]
using `source generated_bash_script` in my `.zshrc` does the job in the meantime

**Comment by DanCardin** [comment]
Sorry about that! I guess the last time I was working on completions, I began renaming the variable, but didn't follow through everywhere. Regenerating the script should, at least, fix the issue I encountered upon copying your above script to test locally.

As of 0.14.1, just released.

---

For future reference (to myself even), `COMPLETION_LINE='toto --' COMPLETION_LOCATION=100 toto --completion=complete` should tell one whether the completions themselves are working, given some example input.

In this case, it was that the script itself was not correct.

Determining the difference between successfully executing completions in which the script is broken, and the script not executing at all is still unknown/tricky; beyond adding "echo" statements to the script itself.
